src/components/Sections/Ourvoice/ContainerRight.js

```diff
@@ -0,0 +1,69 @@
+function ContainerRight() {
```

Comment:
This component need to be refactored.

Reply:
We have to cut this component in small components since the  Let's talk about react isn't an static title.

1. Create a card component.
2. Incrust the card component here (the card componen will be the articles)